dashboard/app/models/pd/workshop.rb

```diff
@@ -81,6 +81,7 @@ class Pd::Workshop < ActiveRecord::Base
       SUBJECT_CS_IN_S_PHASE_3_SEMESTER_2 = 'Phase 3 - Semester 2'
     ],
     COURSE_CSP => [
+      SUBJECT_CSP_SUMMER_WORKSHOP = 'Summer Workshop',
```

Comment:
Sounds like the PLC team is busy today, but I can't imagine it'll be a big problem to have the options in this order.